frontend/src/common/constants/routes.ts

```diff
@@ -4,7 +4,6 @@ export enum ROUTES {
   COLLECTION = "/collections/:id",
   COLLECTIONS = "/collections",
   DATASETS = "/datasets",
-  PRIVATE_COLLECTION = "/collections/:id/private",
```

Comment:
We should probably get someone from the FE team to sanity check the `.ts` changes.